presto-hive/src/main/java/com/facebook/presto/hive/PartitionLoader.java

```diff
@@ -65,4 +74,15 @@ public List<HivePartitionKey> getPartitionKeys(Table table, Optional<Partition>
         }
         return partitionKeys.build();
     }
+
+    protected Map<String, String> buildDirectoryContextProperties(ConnectorSession session)
```

Comment:
should this method be static ?

Reply:
Is there a different more appropriate class to put this utility method.

Reply:
Moved it to HiveUtil